src/neural-network.js

```diff
@@ -366,6 +368,25 @@ export default class NeuralNetwork {
     return true;
   }
 
+
+  /**
+   *
+   * @param {{error: number, iterations: number}} status
+   * @param {[]} data
+   * @private
+   * @returns {{error: number, iterations: number}}
+   */
+  _updateTrainingQuality(status, data){
+    this.trainingQuality.iterations += status.iterations;
+    this.trainingQuality.error = status.error;
+    this.trainingQuality.trainingSets.push({
+      ...status,
+      trainingSetLength: data.length,
+      local: true,
```

Comment:
What does local mean?

Reply:
Probably need to rename this. It differentiates if a training set want used locally or not. Do if you import a net, then all training that had been done previously is deemed non local